src/sentry/static/sentry/app/components/events/groupingInfo.jsx

```diff
@@ -0,0 +1,335 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import _ from 'lodash';
+
+import EventDataSection from 'app/components/events/eventDataSection';
+import SentryTypes from 'app/sentryTypes';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import LoadingError from 'app/components/loadingError';
+import {t} from 'app/locale';
+import KeyValueList from 'app/components/events/interfaces/keyValueList';
+
+import withApi from 'app/utils/withApi';
+
+const StyledGroupVariantList = styled('ul')`
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  font-size: 14px;
+  line-height: 18px;
+`;
+
+const StyledGroupVariantListItem = styled(({contributes, ...props}) => <li {...props} />)`
```

Comment:
You don't need to destruct `contributes` out, styled is smart enough to know that it's not a dom property.

Reply:
Closing this one for the conversation below.